src/crypto/store/base.ts

```diff
@@ -223,6 +223,11 @@ export interface ISession {
     sessionData?: InboundGroupSessionData;
 }
 
+/** Extended data on a Megolm session */
+export interface ISessionExtended extends ISession {
```

Comment:
We don't prefix with I anymore

```suggestion
export interface SessionExtended extends ISession {
```